assets/dev/js/editor/document/elements/commands/copy.js

```diff
@@ -24,7 +24,10 @@ export class Copy extends CommandBase {
 
 		elementorCommon.storage.set(
 			storageKey,
-			containers.map( ( container ) => container.model.toJSON( { copyHtmlCache: true } ) )
+			containers.sort( ( first, second ) => {
```

Comment:
If the problem is purely reverse order, it's better to use `reverse()` instead of `sort()` with expensive DOM probing

Reply:
Actually the problem was not a reversed order. Reversed order is just one of the occurrences, the real problem was that elements were copied by the order they were created, instead of the order in which they are presented in the document.